packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -59,6 +63,58 @@ void main() {
       ProcessManager: () => FakeProcessManager.any(),
     });
 
+    testWithoutContext('wildcard directories do not include subdirectories', () async {
+      final Platform platform = FakePlatform();
+      final FileSystem fs = MemoryFileSystem.test();
```

Comment:
This test failed on ToT and had to be reverted. To make this more stable, you'll need to change this to use testUsingContext and `getFlutterRoot()`